timestep.c

```diff
@@ -32,7 +32,6 @@ void set_global_time(double newtime) {
     All.cf.fac_egy = pow(All.Time, 3 * GAMMA_MINUS1);
     All.cf.hubble = hubble_function(All.Time);
     All.cf.hubble_a2 = All.Time * All.Time * hubble_function(All.Time);
-    All.cf.D1 = GrowthFactor(All.cf.a);
```

Comment:
It is a small change, but useful, because the growth factor used is not quite right (at the level of 1%) in the presence of radiation (or massive neutrinos). So we don't want to use it for anything dynamical.